### ISSUE TITLE
chybová hláška - tábor - načtení akce ze skautISu (před začátkem tábora)

### ISSUE THREAD

**Issue by ellie-RM** [skaut/Skautske-hospodareni#2216]
### Popis chyby
Tábor se zadá ve skautISu a pak když se v modulu tábory načte, je tam tato chybová hláška

![image](https://github.com/skaut/Skautske-hospodareni/assets/133917699/97bdce9a-ad54-4219-a299-09cd3c4d8b6e)

### Jak chybu nasimulovat
Kroky pro nasimulování chyby:
1. tábor zadán do skautISu
2. tábor se otevře v sekci Tábory
3. v sekci základní údaje a evidence plateb se objevuje chybová hláška o chybějící kategorii

pro provoz tábora se dá tato hláška přeskočit (resp. kliknout na Aktivovat dopočítání) , ale pro naše hospodáře táborů je to matoucí; rozpočtové položky zadáváme tak jak to po nás chce skautIS - nevím ale která/é položky se nepárují se skautISem správně

### Očekávané chování
načtení tábora ze skautISu bez chybové hlášky

### Snímky obrazovky
![image](https://github.com/skaut/Skautske-hospodareni/assets/133917699/19661b5f-e45e-458b-8c08-4451235eeaed)

### Uživatelské informace:
 - Sekce: Tábory/základní údaje a evidence plateb
 - Uživatelská role: středisko hospodář (ale i správce akce středisko, vedoucí oddílu)
 - Tábor: https://h.skauting.cz/tabory/13761

**Comment by sinacek** [comment]
> dopočítání funguje, to ano,
Kdyz aktivujes dopocitavani tak ta hlaska zmizi? měla by.

> řeším, že vůbec jako uživatel nevím proč tam ta hláška je, stejně s ní nic neudělám a tak to odkliknu - nevím co to udělalo - systém mi nic nenapíše
Nevím jak jinak to řešit moc jinak. Je to operace kterou je poteba udělat, aby se vygenerovaly kategorie a změní to přepínač ve skautisu myslim, že tento

<img width="513" alt="Screenshot 2023-05-22 at 11 07 48" src="https://github.com/skaut/Skautske-hospodareni/assets/1140123/c6c7aed4-c3b6-4b41-80a7-1bc102bce7fe">

**Comment by ellie-RM** [comment]
ano hláška zmizí

možná by pomohla tedy jiná citace nebo z toho uživatele vynechat? ;D tedy aby se dělo na pozadí
- co přesně potřebuješ při převodu ze skautISu udělat? třeba mě po uživatelské rovině napadne méně matoucí hláška

**Comment by sinacek** [comment]
Potřebuju změnit ten přepínač jsem jsem poslal printscreen. Co všechno na to je a může být ve skautisu navázáno nedokážu říct. Mě jde jen o aktivaci automatického dopocitavani, tak se snazim drzet textaci podobnou skautisu, ale chapu ze to neni uplně čitelné, jen nemám lepší

**Comment by juradee** [comment]
@sinacek @ellie-RM  Návrh na vylepšní funkcionality

V případě schválení tábora střediskem.
![image](https://github.com/skaut/Skautske-hospodareni/assets/4532277/5c40246a-f2c9-4bf6-94f3-eab9364219b0)

Dokud není tábor schválený střediskem
![image](https://github.com/skaut/Skautske-hospodareni/assets/4532277/37d34973-1255-4ac3-a6cc-37234dea3297)

**Comment by juradee** [comment]
No, je otázka, proč tam tahle informace je, protože na možnost zmáčknout to tlačítko to předtím nemělo žádný vliv

```
    <p>
        Nepodařilo se načíst všechny rozpočtové kategorie.<br />
        {if $isApproved} <i class="far fa-check-circle"></i> {else} <i class="far fa-times-circle"></i> {/if} Je tábor schválen střediskovou radou?<br />
        {if $isEditable} <i class="far fa-check-circle"></i> {else} <i class="far fa-times-circle"></i> {/if} Máte oprávnění upravovat tábor?<br /><br />
        <a n:href="activate!" class="btn btn-primary{if !$canActivate} disabled{/if}">Aktivovat dopočítávání</a>
    </p>
```

Je možné, že mi něco uniká

**Comment by sinacek** [comment]
Myslim ze kdyz jsi na to nemel prava tak to spadlo a nic neuldelalo

**Comment by juradee** [comment]
@sinacek asi bych potřeboval příklad, kdy se toto stane. Nedaří se mi replikovat stav, kdy můžu do administrace tábora a přitom nemám zmíněné oprávnění

**Comment by sinacek** [comment]
Jako Aktivní činnovník střediska můžu do administace a nemůžu ale změnit počítání viz obrázek

<img width="1367" alt="Screenshot 2023-06-21 at 19 27 54" src="https://github.com/skaut/Skautske-hospodareni/assets/1140123/1a52606a-3fd1-4545-92cd-d21674c1898f">

**Comment by ellie-RM** [comment]
za mě by tam mohla být i ta větev - a klidně s tím nemáš práva pro editaci tábora (just for sure)
tato textace je už pro uživatele pochopitelná

**Comment by juradee** [comment]
Pokusím se to replikovat. Pokud k tomu uživatel nemá oprávnění, tak bych mu tuto informaci poskytnul. Tlačítko pak nebude mít dostupné a uvidí pouze to, co může.
